docs/migration/Version_0.3.4_0.4.0.md

```diff
@@ -1,4 +1,4 @@
-# Migration from 0.3.3 to 0.3.4
+# Migration from 0.3.3 to 0.4.0
```

Comment:
Why are you overriding this migration guide? Shouldn't we create a new one: `0.3.4 to 0.4.0`? And also this I would see in another PR, not in this one.

Reply:
It was wrong the title compared to the file name. I have to adjust also from `0.3.4`